automol/util/vector.py

```diff
@@ -121,19 +119,19 @@ def best_unit_perpendicular(xyzs):
     elif len(xyzs) == 3:
         nvec = unit_perpendicular(*xyzs)
     else:
-        xyz0 = numpy.sum(xyzs, axis=0) / len(xyzs)
-        _, _, vmat = numpy.linalg.svd(numpy.subtract(xyzs, xyz0))
+        xyz0 = np.sum(xyzs, axis=0) / len(xyzs)
+        _, _, vmat = np.linalg.svd(np.subtract(xyzs, xyz0))
         nvec = vmat[2, :]
 
     nvec = tuple(map(float, nvec))
```

Comment:
Delete this line (see above), since we are no longer converting to `tuple[float, float, float]`.

Reply:
Do i get rid of the whole else statement?

Reply:
No, only delete the following line:
```
nvec = tuple(map(float, nvec))
```
This is converting `nvec` from an `ndarray` to a tuple.